piwik/piwik
# Show warning when paq.push is not initialized correctly

refs https://github.com/piwik/piwik/issues/10878

This is PR for Piwik 3 for https://github.com/piwik/piwik/pull/10888

To test use eg 
```html
<html>
<head></head>
<body>
<script type="text/javascript" src="//apache.piwik/piwik.js"></script>
</body>
</html>
```

Then initialize tracker after document was loaded via `_paq.push`. You will see a console message and no created tracker. When using _paq.push for tracking, the Tracker needs to be configured upfront. Otherwise bugs may occur.
